### ISSUE TITLE
Error on arm build GD5

### ISSUE THREAD

**Issue by SnailMail1** [4ian/GDevelop#700]
I ran the build on a raspberry pi 2b with the latest Raspbian iso dated from the 9th of October. The build failed with the error "Unsupported arch arm" (see log below). I found the following git [post](https://github.com/electron-userland/electron-builder/issues/1478) referring to the same issue. I am out of my depth if anyone has suggestions I am all ears. 

[ArmBuildErrorLog.txt](https://github.com/4ian/GDevelop/files/2484737/ArmBuildErrorLog.txt)
![errorraspberrypibuild](https://user-images.githubusercontent.com/44212084/47044284-c1e56480-d15d-11e8-9be4-4debe1c55d71.png)

**Comment by 4ian** [comment]
I never tried building on arm and it seems that as mentionned in the post that you linked arm is not supported.
What is the exact command line that you've launched? Can you paste it here?

Maybe you should try to build a simple "tar.gz" as the target first (I assume that you tried Appimage according to the post you linked)?

**Comment by SnailMail1** [comment]
I tried yarn build --armv71 --linux tar.gz  - with the error unknown argument armv71.

I attached the log from the terminal.
[arm build error.txt](https://github.com/4ian/GDevelop/files/2492888/arm.build.error.txt)

**Comment by 4ian** [comment]
Where have you found this argument armv71? :/ It's not supported by electron builder.

Can you try just `yarn build --linux tar.gz`?

**Comment by SnailMail1** [comment]
With that command I get the error unsupported arch arm.

**Comment by SnailMail1** [comment]
Yup you are correct somewhere along the way I changed the l to a 1. Switching it back fixed the issue. I am able to build the tar.gz and appimage. From what I can tell both builds appear to be fully functional.

**Comment by 4ian** [comment]
Good to know! Thanks @Wend1go for finding the problem :)

**Comment by SnailMail1** [comment]
If anyone wants to build for raspberry pi as well the two commands that worked for me were - 
yarn build --linux tar.gz --armv7l  this one had errors that dependencies were missing but everything appears to run fine.

The other was the same only change was appimage - I am not sure how those are supposed to build but the files produced appear to be the same as the tar.gz

Thank you @Wend1go and @4ian 

![2018-10-19-191823_1360x768_scrot](https://user-images.githubusercontent.com/44212084/47260029-80b4c380-d481-11e8-9437-508d2f63005b.png)
![2018-10-20-092133_1360x768_scrot](https://user-images.githubusercontent.com/44212084/47260030-80b4c380-d481-11e8-9c91-250d8a0a634e.png)

**Comment by SnailMail1** [comment]
For the most part it does, the main start window takes about 20 seconds to come up. Once you open a project it is relatively quick, I made a basic project creating animations and it all worked. The built in drawing tool came up pretty quick along with all other menu systems. The only part that was noticably slow was the preview, the few examples I tried ( platform, zombie laser) ran at maybe 10 to 15 fps.
